### PR TITLE
Us24 user page

### PR DESCRIPTION
**Pull request by Adwarf14**

### CODE REVIEW

**Review by Gumatan — DISMISSED** [review]

File: backend/index.js [hunk 1]
```diff
@@ -36,26 +37,52 @@ app.get("/api/users", (req, res) => {
...
+
+app.get("/api/user/posts/:id", (req, res) => {
   db.query(
-    "SELECT post.id, circle_id, user_id, user_id_team, game_id, message, date, image_url, user.avatar AS user_avatar from post JOIN user on user_id=user.id ORDER BY id DESC LIMIT 10 OFFSET ?  ",
-    [Number(req.params.limit)],
+    "SELECT post.id, user_id, user_id_team, game_id, message, date, image_url from post WHERE post.user_id = ?",
+    [req.params.id],
     (err, results) => {
       if (err) {
+        console.log(err);
```
Comment (LoicBrassart) [inline backend/index.js]: No console.log

File: backend/index.js [hunk 2]
```diff
@@ -36,26 +37,52 @@ app.get("/api/users", (req, res) => {
...
         res.status(500).send(err);
       } else {
+        console.log(req.query.id);
```
Comment (LoicBrassart) [inline backend/index.js]: No console.log

File: backend/index.js [hunk 3]
```diff
@@ -36,26 +37,52 @@ app.get("/api/users", (req, res) => {
...
-app.get("/api/gamelist/", (req, res) => {
+// FIL-ACTU || GET & POST
+app.get("/api/posts/:limit", (req, res) => {
   db.query(
-    "SELECT * from game",
+    "SELECT id, circle_id, user_id, user_id_team, game_id, message, date, image_url from post ORDER BY id DESC LIMIT 10 OFFSET ?  ",
+    [Number(req.params.limit)],
     (err, results) => {
       if (err) {
+        console.log(err);
```
Comment (LoicBrassart) [inline backend/index.js]: No console.log

File: backend/index.js [hunk 4]
```diff
@@ -88,6 +125,7 @@ app.post("/api/posts", (req, res) => {
 })
 
 app.listen(backendPort, err => {
+  console.log(backendPort + "Marche");
```
Comment (LoicBrassart) [inline backend/index.js]: No console.log

File: frontend/src/App.js [hunk 5]
```diff
@@ -4,10 +4,11 @@ import { useSelector } from "react-redux";
 import NavBar from "./components/NavBar";
 import Carousel from "./components/Carousel";
 import SignIn from "./components/SignIn";
-import Form from "./components/Form";
 import NewsFeed from "./components/NewsFeed";
+import UserPage from "./components/UserPage";
 import ContainerEditProfile from "./components/EditProfile/ContainerEditProfile";
 
+
```
Comment (LoicBrassart) [inline frontend/src/App.js]: (This second empty lines makes me think your Prettier is misconfigured and will wreck havoc in the project)

File: frontend/src/store/reducer.js [hunk 6]
```diff
@@ -1,27 +1,28 @@
 const initialState = {
-    offsetPosts: 0,
-    jwt: null,
-    reload: 0
+  offsetPosts: 0,
```
Comment (LoicBrassart) [inline frontend/src/store/reducer.js]: (I'm surer and surer: your Prettier doesn't check `.js` files... Please ask someone for their conf, or ask it from me if needed)

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Some upkeep to do

**Review by Gumatan — DISMISSED** [review]

**Pull request closed without merging**